tests/main/views/test_buyers.py

```diff
@@ -1653,6 +1653,17 @@ def test_404_if_brief_does_not_belong_to_user(self, data_api_client):
 
 @mock.patch('app.main.views.buyers.data_api_client', autospec=True)
 class TestBriefSummaryPage(BaseApplicationTest):
+
```

Comment:
Not a blocker but I think these tests could be a bit more readable with something like this

```
SIDE_LINKS_XPATH = '//div[@class="column-one-third"]//a'
INSTRUCTION_LINKS_XPATH = '//main[@id="content"]//ul/li/a'

@staticmethod
def _get_links(document, xpath, text_only=False):
    if text_only:
        return [e.text_content() for e in document.xpath('//main[@id="content"]//ul/li/a')]
    else:
        return [(e.text_content(), e.get('href')) for e in document.xpath('//div[@class="column-one-third"]//a')]

def test_foo(self, dataapiclient):
    ...
    assert self._get_links(document, self.SIDE_LINKS_XPATH) == [('Foo', 'my/cool/foo/url')]
```

The above assertion tells us that we're getting the side links from the document using the xpath and that they should match the list we've defined. We don't really need to go and look at what `_get_links` does because it's super obvious from the name and signature. I prefer not to extrapolate too much out in tests because it breaks the flow of reading a test if you have to keep jumping around to different methods.

This is personal preference though. Approving now.

Reply:
I like the idea of making it clear we're using xpath, thanks! Also one generic helper is always better than two 😸